drivers/w3s.go

```diff
@@ -365,7 +365,7 @@ func runWithCredentials(cmd *exec.Cmd, proof string) ([]byte, error) {
 }
 
 func base64UrlToBase64(proof string) (string, error) {
-	ucanProofByte, err := base64.URLEncoding.DecodeString(proof)
+	ucanProofByte, err := base64.URLEncoding.WithPadding(base64.NoPadding).DecodeString(proof)
```

Comment:
WDYT of redefining this (before DecodeString) as a global var in this file?

Reply:
yes, good idea.